haystack/core/pipeline/pipeline.py

```diff
@@ -171,13 +353,6 @@ def run(self, word: str):
         # Initialize the inputs state
         components_inputs: Dict[str, Dict[str, Any]] = self._init_inputs_state(data)
```

Comment:
Seeing how we do more input init below, we should rename this function to be more exact.